### ISSUE TITLE
Can't click yes or no on UAC window. 

### ISSUE THREAD

**Issue by TheHaso** [Tylemagne/Gopher360#25]
I'm using the latest gopher and X1 Controller, whenever I launch a game that requires admin rights, I can't click yes on the window. The mouse just hoovers the button. 

Using the pc mouse I'm able to do it, but weird it does not apply for the mouse controlled with gopher. 

I'm running gopher as admin yes.

**Comment by Naseband** [comment]
No clue if it can be "fixed" or worked around, but it's not a bug.
I guess it's a security feature, otherwise any (malicious) program could easily skip those messages, maybe even without your notice.

**Comment by Alphabetix** [comment]
This could be "solved" by converting gopher into a driver instead of an external program.

**Comment by TheHaso** [comment]
An such implement would be pretty nice. Its so neat to have pc/laptop connected to the TV, but everytime a program/game gives UAC promt, one have to go up and noe it manually.

**Comment by sighol** [comment]
Would it be possible to turn off the UAC instead?

@alphabetix I have never developed a windows driver before. I suspect that converting the program to a driver will require a big rewrite. If you are up for the task, you could fork the project and I'll add a section in the Readme with a link to the fork. I'm sorry that I can't be of more help.

**Comment by TheHaso** [comment]
@sighol 

Yes is it possible to turn off UAC, and this will temporally solve the issue, but leave the user for huge security risk (personal experience, a virus exe that was able to execute itself and duplicate to every connected device automatically). Therefore I lower UAC one step to disable black background at UAC.

**Comment by Zero3K** [comment]
Any news regarding the fixing of this issue?

**Comment by YahiaDru001** [comment]
this is ain't the only window which can't use the Controller to move when its shown , I mean 
some apps makes the same issue , eg. : when I open the device manager , same problem 
u can't move the mouse , another program :  MSI AfterBurner 

so I think the issue can be fixed & it's not a security access feature 

Thnx , waitin' for a help

**Comment by shad00m** [comment]
I can understand that the UAC "fix" will not come as the whole driver zibbityboo seems pretty hard to do. Although I've noticed every "android remote mouse" seems to install a driver that works fine with UAC.
I tried "Unified Remote" but it can screenshare, access my files and calls back home every now and then...

Can someone recommend me a android remote? A simple app that i can click "yes" on the UAC for or any other workaround is welcome (except turning UAC off)
